doc/sphinx/changes.rst

```diff
@@ -14,6 +14,85 @@ Version 8.17
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+Coq version 8.17 integrate a soudness fix to the Coq kernel along
+with a few new features and a host of improvements to the Ltac2 language
+and libraries. We highlight some of the most impactful changes here:
+
+  - :ref:`Fixed <817VmCompute>` a logical inconsistency due to :tacn:`vm_compute` in
+    presence of side-effects in the enviroment (e.g. using `Back` or `Fail`).
+
+  - It is now possible to dynamically :ref:`enable or disable <817Notations>` notations.
+
+  - Support :ref:`multiple scopes <817Scopes>`` in :cmd:`Arguments` and :cmd:`Bind Scope`.
+
+  - The tactics chapter of the manual has seen :ref:`many improvements <817TacticsRefman>`
+    in presentation and wording and is semi-automatically checked for coherence with the implementation.
+
+  - :ref:`Fixes <817Eauto>` to the `auto` and `eauto` tactics, to respect hint priorities and the documented use
+    of :tacn:`simple apply`. Potentially breaking change.
+
+  - :ref:`New LTac2 <817Ltac2>` APIs, deep pattern-matching with ``as`` clauses and handling of literals,
+    support for record types and preterms.
+
+  - :ref:`Move <817ClassFieldSyntax>` from :g:`:>`: to :g:`::` syntax for declaring typeclass fields as instances, fixing
+    a confusion with declaration of coercions.
+
+  - :ref:`Standard library <817Stdlib>` improvements.
+
+See the `Changes in 8.17.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.17 <https://coq.github.io/doc/v8.17/refman>`_,
+`documentation of the 8.17 standard library <https://coq.github.io/doc/v8.17/stdlib>`_
+and `developer documentation of the 8.17 ML API <https://coq.github.io/doc/v8.17/api>`_
+are also available.
+
+Ali Caglayan, Emilio Jesús Gallego Arias, Gaëtan Gilbert
+and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+Erik Martin-Dorel has maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+Maxime Dénès, Paulo G. Giarrusso, Huỳnh Trần Khanh, and Laurent Théry have
+maintained the VsCoq extension for VS Code.
+
+The OPAM repository for Coq packages has been maintained by
```

Comment:
```suggestion
The opam repository for Coq packages has been maintained by
```
The opam [website](https://opam.ocaml.org) itself uses "opam" instead of OPAM, so it may be time to change.
> opam is a source-based package manager for OCaml.